README.md

```diff
@@ -5,7 +5,17 @@ Typically, the left-hand cohorts are several weekly or monthly extracts of pseud
 The right-hand cohort is a single extract of these data for variables that are not expected to change.
 Extracting and joining the cohorts in this way is more efficient than several weekly or monthly extracts of these data for all variables.
 
-[left join]: https://en.wikipedia.org/wiki/Join_(SQL)#Left_outer_join
+Using cohort-joiner has several advantages over writing a [scripted action][]:
+
+* cohort-joiner uses the same logic as cohort-extractor to save files,
```

Comment:
nice! all advantages are clear to me. we could highlight the main point of each advantage by adding a "heading before describing it e.g.,

- Compatibility: ...
- Efficiency: ...
- Transparency: ...

Reply:
Don't forget the [`suggestion` block][1]! I've summarized each advantage, as you suggest, in 4d8d87e.

[1]: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request